MonkeyBot/Initializer.cs

```diff
@@ -27,7 +27,7 @@ public static async Task<IServiceProvider> InitializeAsync(ApplicationArguments
             });
 
             DiscordSocketClient client = services.GetService<DiscordSocketClient>();
-            string token = (await DiscordClientConfiguration.LoadAsync().ConfigureAwait(false)).Token;
+            string? token = (await DiscordClientConfiguration.LoadAsync().ConfigureAwait(false)).Token;
             await client.LoginAsync(TokenType.Bot, token).ConfigureAwait(false);
```

Comment:
maybe we should assert the token first before trying to Log In (neither null nor empty string)

Reply:
Agreed

Reply:
ErrorMessage is only set when the Precondition is actually checked before running a command if I interpret it correctly. It defaults to null. 
But just leave as is, I just looked at the whole method and it just asks for a refactoring to a switch expression anyway :)

Reply:
I think it throws when it can't log in but I will have a look after merging. It's a bit off scope here